### DISCUSSION TITLE
Is the docker nautobot development only?

### DISCUSSION THREAD

**Discussion by adamantcoffeemug** [nautobot/nautobot#6458] [Q&A]
Hello everyone, 

I've been installing nautobot through containers and I doesn't seem to accept https or a production environment context. 

```
docker container logs server_c | grep -i http
  "GET / HTTP/1.1" 302 0
  You're accessing the development server over HTTPS, but it only supports HTTP.
```

Documentation doesn't mention production environment, https://docs.nautobot.com/projects/secrets-providers/en/latest/dev/dev_environment/ 

Is there something I'm missing? Is there a reference for production context nautobot_config.py? 

Thanks in advance,

**Comment by itdependsnetworks** [comment]
Not 100% sure of the exact question you are asking, but I think the below context should help

* We publish containers with Nautobot installed in them: https://hub.docker.com/r/networktocode/nautobot
* The linked documentation is for local development environment, so certainly not intended for production
* We publish helm charts which is more intended for production deployment with Kubernetes https://github.com/nautobot/helm-charts
* The nautobot_config.py is configured based on your environment, so there is no official all encompassing file

Often, these convo's are best for our public slack, sign up [here](http://slack.networktocode.com/), make sure to join the #nautobot channel.

**Reply by adamantcoffeemug** [reply]
> Not 100% sure of the exact question you are asking, but I think the below context should help

My main question is how do I active HTTPS on the nautobot container, because besides that, everything seems to be functionning. 
The web browser states that the page has no certificate and the logs confirm it doesn't support https. 
The command that starts the nautobot servers is NOT done with "--insecure"

So I'm a bit confused about what I'm missing on the ssl configuration of the web interface. 
The LDAP integration too, but that's not my main concern right now.

**Reply by jvanderaa** [reply]
You should be serving the HTTPS from an NGINX or similar container that connects to the web app of Nautobot. You may want to look at the Nautobot Docker Compose library as a method to help. https://github.com/nautobot/nautobot-docker-compose

**Comment by adamantcoffeemug** [comment]
Colleague gave up on the projet, nothing to help now.
